pyseq.py

```diff
@@ -733,7 +805,7 @@ def getSequences(source):
         items = sorted(source, key=lambda x: str(x))
     elif isinstance(source, str):
         if os.path.isdir(source):
-            items = sorted(glob(os.path.join(source, '*')))
+            items = sorted(glob(os.path.join(source, '*.*')))
```

Comment:
I have been using this to get rid of folders from getSequences.
It should only find files with an extension *.* ...
In my fork I had this with a keyword (folder=False)
    elif type(source) == str and os.path.isdir(source) and folders:
        items = sorted(glob(os.path.join(source, '*')))
    elif type(source) == str and os.path.isdir(source) and not folders:
        items = sorted(glob(os.path.join(source, '*.*')))

but maybe it is better than to filter folders out specifically instead of globbing only for files with a .

I ll remove that for now.